client-api/src/main/java/io/smallrye/graphql/client/typesafe/api/GraphQlClientBuilder.java

```diff
@@ -0,0 +1,36 @@
+package io.smallrye.graphql.client.typesafe.api;
+
+import java.net.URI;
+import java.util.Iterator;
+import java.util.ServiceConfigurationError;
+import java.util.ServiceLoader;
+
+public interface GraphQlClientBuilder {
+
+    static GraphQlClientBuilder newBuilder() {
+        ServiceLoader<GraphQlClientBuilder> loader = ServiceLoader.load(GraphQlClientBuilder.class);
```

Comment:
Does RESTClient also use SPI ?

Reply:
yes. it didn't before, but as the api is in a different module, it was necessary.